api/src/OpenTelemetry/Logs/Core.hs

```diff
@@ -124,41 +126,56 @@ setGlobalLoggerProvider :: (MonadIO m) => LoggerProvider body -> m ()
 setGlobalLoggerProvider = liftIO . writeIORef globalLoggerProvider
 
 
+defaultShutdownTimeout :: Int
+defaultShutdownTimeout = 5_000_000
```

Comment:
`shutdown` previously never timed out. I used the same shutdown timeout as `forceFlush`, but there is no reason that 5,000,000 microseconds is a magic number. There might be a better default timeout.